SixTest/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+
+ADD_TEST(NAME SelfTest_checkf10 COMMAND checkf10)
+set_tests_properties(SelfTest_checkf10 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*checkf10_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_compf10 COMMAND compf10)
+set_tests_properties(SelfTest_compf10 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*compf10_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_checkf110 COMMAND checkf110)
+set_tests_properties(SelfTest_checkf110 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_checkf1014 COMMAND checkf1014)
+set_tests_properties(SelfTest_checkf1014 PROPERTIES PASS_REGULAR_EXPRESSION "Comparing VERSION    0.0000000000000000 *to *0.0000000000000000.*\n.*checkf1014_DIFF I\\/O error, wrong no of lines\\!\\! line no *0")
+
+ADD_TEST(NAME SelfTest_read90 COMMAND read90)
+set_tests_properties(SelfTest_read90 PROPERTIES PASS_REGULAR_EXPRESSION "At line .* of file .*read90.f90 \\(unit = 190\\)\nFortran runtime error: Cannot open file 'fort.190': No such file or directory")
+
+SET(SIXTRACK_TESTS ${SIXTRACK_TESTS})
+#SET(SIXTRACK_TESTS bbe51 bbe52 bbe571ib0 bb bb_ntwin1 crabamp dipedge distance dynk_globalvars elensidealthck4d elensidealthck6d elensidealthin4d elensidealthin6d_DYNK elensidealthin6d eric exact fma frs60 frs javier lostevery lost lostnotilt notilt prob1 prob3 s316 thick4 thick6dblocks thick6ddynk tilt ${SIXTRACK_TESTS})
+
+##
+## Needs checking: thick6dsingles (missing files)
+## thick6dblocks elensidealthck6d dynk_globalvars thick6ddynk - empty fort.10
+## Disabled: elensidealthin6d_DYNK
+## "Totally untested": bnl
+##
+
+SET(STF_TESTS bbe51 bbe52 bbe571ib0 bb_ntwin1 bb crabamp dipedge distance elensidealthck4d elensidealthin4d elensidealthin6d eric exact fma frs60 frs javier_bignpart javier lostevery lostnotilt lost notilt prob1 prob3 s316 thick4 tilt)
+
+#only turn on the bnl test if we have BNLELENS enabled due to slight differences in the test
+if(BNLELENS)
+	SET(SIXTRACK_TESTS bnl ${SIXTRACK_TESTS})
+endif(BNLELENS)
+
+if(STF AND (BIGNPART OR HUGENPART))
+	SET(SIXTRACK_TESTS javier_bignpart ${SIXTRACK_TESTS})
+endif()
+
+if(NOT COLLIMAT)
+	SET(SIXTRACK_TESTS bbe51 bbe52 bbe571ib0 bb bb_ntwin1 crabamp dipedge distance dynk_globalvars elensidealthin4d elensidealthin6d eric exact frs60 frs javier lostevery lost lostnumxv lostnotilt notilt prob1 prob3 s316 thick4 thick6dblocks tilt elensidealthck4d elensidealthck6d fma thick6ddynk ${SIXTRACK_TESTS})
+endif(NOT COLLIMAT)
+
+#For CR builds, test the CR functions, so kill/resume the test
+if(CR)
+	SET(CR_ENABLE 1)
+	SET(CR_KILL_TIME 10)
```

Comment:
What happens when you try and run a CR binary on a run that has already finished? Will it just do nothing or re-run everything from the start?